CHANGES.rst

```diff
@@ -2,6 +2,13 @@
 Changelog
 =========
 
+
+Unreleased
+==========
+
+- Add documentation about TinyMCE styles and formats [davilima6]
+
+
```

Comment:
I agree, however I'm not sure this change is smaller than some from previous release. Would you prefer me to remove it from changelog at this time?